src/components/scenes/Loans/ManageCollateralScene.tsx

```diff
@@ -307,11 +308,12 @@ export const ManageCollateralScene = <T extends keyof ParamList>(props: Props<T>
         />
         {showAprChange ? <AprCard apr={newDebtApr} key="apr" /> : null}
         <EdgeText style={styles.textTitle}>{opTypeStringMap[actionOpType].srcDestCard}</EdgeText>
-        {bankAccountsMap != null ? (
-          <TappableAccountCard emptyLabel={s.strings.loan_select_receiving_wallet} selectedAsset={selectedAsset} onPress={handleShowWalletPickerModal} />
-        ) : (
-          <FillLoader />
-        )}
+        <Space around={0.5}>
```

Comment:
Why is additional margin of 0.5 needed where it wasn't there before?

Reply:
It was removed from the child component: https://github.com/EdgeApp/edge-react-gui/pull/3683/files/682b5690268c845107510fa92bf4e1995c733cd8..fbe9c3f360e02e1e5cf7745ca8b7816692e23b13#diff-df9908d5cd34455d02b9dfc1d05d1ddb5e4923364bb0e7cb2d58dd6b9e3cbb45R46